docs/subnets/commit-reveal.md

```diff
@@ -32,30 +64,22 @@ style={{width: 750}}
 />
 </center>
 
-<br />
-
-1. A subnet validator sets the weights normally by using [`set_weights`](pathname:///python-api/html/autoapi/bittensor/core/extrinsics/set_weights/index.html). 
-2. But behind the scenes, the commit reveal feature will use an internal method called [`commit_weights`](pathname:///python-api/html/autoapi/bittensor/core/extrinsics/commit_weights/index.html) to commit an encrypted hash of these consensus weights to the blockchain, instead of submitting weights openly to the chain where they can be seen by anyone.
-3. The commit reveal feature then waits for a certain interval. A subnet owner can control this interval by setting the subnet hyperparameter `commit_reveal_weights_interval`.
-4. After this interval is elapsed, the commit reveal automatically reveals these weights by submitting them again, but now openly for everyone to see, to the blockchain.
-5. On the blockchain, the committed hash from step 2 is compared to the hash of the openly submitted (revealed) weights from step 4. If the hashes are the same, the blockchain will apply the weights to the Yuma Consensus algorithm on-chain. If the hashes are not the same, the blockchain will issue an error and the weights are not applied. 
-
-:::tip Commit reveal works behind the scenes
-After the subnet owner turns ON the commit reveal feature, everything happens behind the scenes. A subnet validator will continue to set weights normally by using [`set_weights`](pathname:///python-api/html/autoapi/bittensor/core/extrinsics/set_weights/index.html).
-:::
-
----
 
-## How to use commit reveal feature
+## How to use the commit reveal feature
 
-If you are a subnet owner, set the below hyperparameters to use the commit reveal feature:
+As a subnet owner, set the below hyperparameters to use the commit reveal feature:
```

Comment:
Might be good to specify what minimum version of the python API needs to be used to support this feature.